Builds/rpm/rippled.spec

```diff
@@ -20,13 +24,11 @@ Rippled is the server component of the Ripple network.
 
 
 %prep
-%setup -n rippled-release
+%setup -n rippled-%{rippled_branch}
 
 
 %build
-# Assume boost is manually installed
-export RIPPLED_BOOST_HOME=/usr/local/boost_1_55_0
```

Comment:
Boost is installed and linked in such a way that `RIPPLED_BOOST_HOME` isn't needed.
https://github.com/wilsonianb/rippled/blob/build-rpm/Builds/rpm/Dockerfile#L9-L14